### PR TITLE
Atlassian page

### PR DESCRIPTION
**Pull request by JosueRodriguezGarcia**

### CODE REVIEW

File: src/main/java/core/selenium/WebDriverConfig.java [hunk 1]
```diff
@@ -78,7 +78,7 @@ public long getImplicitlyWaitTime() {
      *
      * @return a explicitly wait.
      */
-    public long getExplicitlWaitTime() {
+    public long getExplicitlyWaitTime() {
```
Comment (MelissaRoman06) [inline src/main/java/core/selenium/WebDriverConfig.java]: Please fix the method name.

File: src/main/java/core/selenium/WebDriverManager.java [hunk 2]
```diff
@@ -55,7 +55,7 @@ private void initializes() {
         this.webDriver.manage()
                 .timeouts()
                 .implicitlyWait(WebDriverConfig.getInstance().getImplicitlyWaitTime(), TimeUnit.SECONDS);
-        webDriverWait = new WebDriverWait(webDriver, WebDriverConfig.getInstance().getExplicitlWaitTime(),
+        webDriverWait = new WebDriverWait(webDriver, WebDriverConfig.getInstance().getExplicitlyWaitTime(),
```
Comment (MelissaRoman06) [inline src/main/java/core/selenium/WebDriverManager.java]: Please fix the method name.

File: src/main/java/core/selenium/util/WebDriverMethod.java [hunk 3]
```diff
@@ -14,6 +14,7 @@
 
 import org.openqa.selenium.WebElement;
 
+
```
Comment (MelissaRoman06) [inline src/main/java/core/selenium/util/WebDriverMethod.java]: Unnecessary empty line.

Comment (JosueRodriguezGarcia) [reply src/main/java/core/selenium/util/WebDriverMethod.java]: Solved

File: src/main/java/core/selenium/WebDriverManager.java (hunk 2 above)
Comment (JosueRodriguezGarcia) [reply src/main/java/core/selenium/WebDriverManager.java]: Solved

File: src/main/java/core/selenium/WebDriverConfig.java (hunk 1 above)
Comment (JosueRodriguezGarcia) [reply src/main/java/core/selenium/WebDriverConfig.java]: Solved

**Review by MelissaRoman06 — APPROVED** [review]

**Pull request merged**